app/assets/stylesheets/components/_blue-box-campaign.scss

```diff
@@ -0,0 +1,45 @@
+$light-grey: #bfc1c3;
+$blue: #005ea5;
+$white: #ffffff;
+
+.app-c-native-blue-box-campaign__border {
+  display: block;
+  border-top: 1px solid $light-grey;
+  padding: 20px 0 0;
+  text-align: center;
+}
+
+.app-c-native-blue-box-campaign__link {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: $blue;
+  width: 100%;
+  min-height: 280px;
+  color: $white;
+  text-decoration: none;
+  text-align: left;
+  box-sizing: border-box;
+  padding: 15px;
+  font-size: 27px;
+  line-height: 30px;
+  font-weight: bold;
+}
+
+.app-c-native-blue-box-campaign__title {
+  display: block;
+  color: $white;
+
+  &:hover {
```

Comment:
Just an FYI - the focus state when you click on the box doesn't meet accessible color contrast guidelines. You may want to consider changing the text to black on focus.

<img width="328" alt="screen shot 2018-03-27 at 17 01 10" src="https://user-images.githubusercontent.com/29889908/37979418-783c21e2-31e0-11e8-8490-3c5527c4684f.png">